PhysicsTools/PatAlgos/plugins/PATMuonMerger.cc

```diff
@@ -79,9 +98,12 @@ void PatMuonMerger::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
     for (auto& muon : *muons) {
       for (unsigned int i = 0, n = muon.numberOfSourceCandidatePtrs(); i < n; ++i) {
         reco::CandidatePtr ptr = muon.sourceCandidatePtr(i);
-        if (ptr.isNonnull() && ptr == pfCandPtr)
+        if (ptr.isNonnull() && ptr == pfCandPtr){
           isPFMuon = true;
+	  break;
```

Comment:
I see that in the actual file they are aligned correctly.
If you remove the tabs they'll also look correctly aligned in github